package.json

```diff
@@ -69,7 +70,7 @@
     "dist-src": "cp -r ./src ./lib && find ./lib -name '__tests__' -type d -exec rm -r '{}' + && ./node_modules/.bin/babel lib --out-dir lib",
     "livereload": "NODE_ENV='development' ./node_modules/.bin/gulp docs:livereload",
     "npm-publish": "npm publish ./",
-    "test": "jest",
+    "test": "mv .babelrc .babelrc-tmp && jest && mv .babelrc-tmp .babelrc",
```

Comment:
Ok, so for others. When this gets merged and you are developing on tests:
1. Delete the `.bashrc`
2. Create your tests running `./node_modules/.bin/jest`
3. Add the `.bashrc` back and continue with your development

The reason for this is that running the tests will look for the `.bashrc`, but because it runs on a different version of babel, it will fail with the current `.bashrc`.

@rcorral is working on updating jest to use the new babel version